src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -410,6 +410,13 @@ method onMessageEdited*(self: Module, message: MessageDto) =
     message.links
     )
 
+  let messagesIds = self.view.model().findIdsOfTheMessagesWhichRespondedToMessageWithId(message.id)
+  for msgId in messagesIds:
+    # refreshing an item calling `self.view.model().refreshItemWithId(msgId)` doesn't work from some very weird reason
+    # and that would be the correct way of updating item instead sending this signal. We should check this once we refactor
+    # qml part.
```

Comment:
I think it doesn't work because we use `getMessageByIdAsJson` for the reply message and it isn't hooked into the model directly, so it isn't binded.
The only way to make it bind, as far as I see it, would be to add a `replyMsg` property to the MessageItem that would also be a MessageItem, but then it's annoying. Maybe we can try and do it later or find another way, but it's low priority I'd guess.